src/classes/OPP_OpportunityContactRolesTribute_TEST.cls

```diff
@@ -465,96 +431,89 @@ public class OPP_OpportunityContactRolesTribute_TEST {
         insert cons;
 
         Opportunity opp = new Opportunity(
-                AccountId = orgAcct.Id,
-                StageName = closedWonStageName,
-                Name='Donation2',
-                Amount= 1000,
-                CloseDate = date.today(),
-                Honoree_Contact__c = cons[0].Id,
-                Notification_Recipient_Contact__c = cons[1].Id,
-                Primary_Contact__c = cons[2].Id
+            AccountId = orgAcct.Id,
+            StageName = closedWonStageName,
+            Name = 'Donation2',
+            Amount = 1000,
+            CloseDate = Date.today(),
+            Honoree_Contact__c = cons[0].Id,
+            Notification_Recipient_Contact__c = cons[1].Id,
+            Primary_Contact__c = cons[2].Id
         );
 
         Opportunity oldOpp = new Opportunity(
-                AccountId = orgAcct.Id,
-                StageName = closedWonStageName,
-                Name='Donation2',
-                Amount= 1000,
-                CloseDate = date.today(),
-                Honoree_Contact__c = cons[0].Id,
-                Notification_Recipient_Contact__c = cons[1].Id,
-                Primary_Contact__c = cons[2].Id
+            AccountId = orgAcct.Id,
+            StageName = closedWonStageName,
+            Name = 'Donation2',
+            Amount = 1000,
+            CloseDate = Date.today(),
+            Honoree_Contact__c = cons[0].Id,
+            Notification_Recipient_Contact__c = cons[1].Id,
+            Primary_Contact__c = cons[2].Id
         );
 
         Set<Id> honoreeIdSet = new Set<Id>();
 
         Test.startTest();
 
-        // getRole will return a Null, which will cause it to decide this field/test is not in use, same as if role value was not set
-        System.assertEquals (false, OPP_OpportunityContactRoles_TDTM.needsManageOCR(null, null, honoreeIdSet, false, 'NO_FIELD', null));
+        System.assertEquals (false, OPP_OpportunityContactRoles_TDTM.needsManageOCR(null, null, honoreeIdSet, false, 'NO_FIELD', null), 'getRole will return a Null, which will cause it to decide this field/test is not in use, same as if role value was not set');
```

Comment:
I think you just moved the comment into the 3rd parm of the assert. It's not exactly what that parameter should represent, but it's not worth changing again. This parameter is the message generated if the assert fails, which makes it easier to debug. In this case, maybe something like "Expected getRole to return a null for some reason" (honestly I don't understand the text in there currently since getRole() is not the method being called, so I'm having a hard time coming up with what it could be). Maybe it does need changing. Not really sure anymore.

Reply:
I did one more pass on these...you can look when I push the last set of changes after we talk and I confirm any other updates I need to bang out.